projects/ui/src/hooks/beanstalk/useMarketData.ts

```diff
@@ -20,7 +22,8 @@ const useMarketData = () => {
 
   /// Cast query data to BigNumber, etc.
   const listings = useCastApolloQuery<PodListing>(listingsQuery, 'podListings', useCallback((_listing) => castPodListing(_listing, harvestableIndex), [harvestableIndex]), loading);
-  const orders   = useCastApolloQuery<PodOrder>(ordersQuery, 'podOrders', castPodOrder, loading);
+  let orders   = useCastApolloQuery<PodOrder>(ordersQuery, 'podOrders', castPodOrder, loading);
+  orders = orders?.filter((order) => order.beanAmountRemaining.gt(MIN_POD_AMOUNT));
```

Comment:
@silochad since `beanAmountRemaining` is something we calculate, not sure if the filter is possible during the query

Reply:
This is a great point. @cujowolf In order to let users query by the number of Pods remaining in an order, they'll need an explicit field on the entity (AFAIK there is no way to query where `beanAmount - beanAmountFilled > X` for example.